src/CountryPicker.js

```diff
@@ -293,17 +289,23 @@ export default class CountryPicker extends Component {
     })
   }
 
-  renderCountry(country, index) {
+  renderCountry = (country, index) => {
+    if (this.disabledCountriesByCode[country]) {
+      return (
+        <View key={index}>
+          {this.renderCountryDetail(country)}
+        </View>
+      );
+    }
     return (
       <TouchableOpacity
         key={index}
         onPress={() => this.onSelectCountry(country)}
-        activeOpacity={0.99}
```

Comment:
Why did they do it? :D

Reply:
don't know this is like TouchableWithOutFeedback;)